### DISCUSSION TITLE
[V5]我想在点击某个Node时获取和他相连的Edge

### DISCUSSION THREAD

**Discussion by xplloveyxl** [antvis/G6#6453] [Q&A] [answered]
我想在点击某个Node时获取和他相连的Edge、要用哪个方法。我找了好久都没找到，求大佬们帮忙

**Comment by uhobnil** [comment, marked as answer]
getRelatedEdgesData ?

**Reply by yvonneyx** [reply]
https://g6.antv.antgroup.com/api/graph/method#graphgetrelatededgesdataid-direction

**Reply by xplloveyxl** [reply]
谢谢大佬们解答，这个是获取数据，但是我想要的是Edge的实例，有什么好方法没，我没找到

**Reply by Aarebecca** [reply]
> 谢谢大佬们解答，这个是获取数据，但是我想要的是Edge的实例，有什么好方法没，我没找到

事实上大多数情况下你不应该获取 Edge 实例，除非你非常了解你要做什么
